app/assets/javascripts/host_edit.js

```diff
@@ -137,6 +137,10 @@ function submit_with_all_params(){
   clear_errors();
   animate_progress();
 
+  // trigger turbolinks event
+  // see webpack/.../MountingService.js
+  document.trigger(new Event('page:before-unload'));
```

Comment:
I think `new Event...` doesn't work without babel for all browsers, you can use jquery `.trigger`